src/main/java/org/gradle/profiler/studio/StudioGradleClient.java

```diff
@@ -103,8 +105,9 @@ public void close() {
             System.out.println("* Android Studio stopped.");
         } catch (Exception e) {
             e.printStackTrace();
-            System.out.println("* Android Studio did not stopped successfully, you will have to close it manually.");
+            System.out.println("* Android Studio did not finish successfully, you will have to close it manually.");
         } finally {
+            tryClose(studioPluginInstaller::uninstallPlugin);
             tryClose(studioPluginServer);
             tryClose(studioAgentServer);
```

Comment:
BTW, we should use try-with-resources (what's a better name for this construct?) here:

https://github.com/gradle/gradle-profiler/blob/c81fc08884fe24a1403d04c390a5ed524ebb0df7/src/main/java/org/gradle/profiler/GradleScenarioInvoker.java#L79